tests/unittests/enable_debug_logging_functions/debug_logging/main.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why add this function? Can we just reuse the functions in log_filtering_functions directory?

Reply:
good catch, will remove